react-frontend/src/index.css

```diff
@@ -92,3 +92,17 @@ body {
 .sidebar-heading {
   text-transform: uppercase;
 }
+
+.table-image {
+  width: 30px;
```

Comment:
Is there a better way in JS to handle these kinds of "magic numbers" so things like image size and border color are more easily configurable?

Reply:
If not, LGTM!

Reply:
Generally the way to abstract away from magic numbers like this in the CSS file would be to use Sass, however given the size of this project I don't think that is necessary. If we want to use that in the future, its an option, but outside of the scope of this feature.